frontend/controller/serviceworkers/sw-primary.js

```diff
@@ -569,3 +569,5 @@ sbp('okTurtles.events/on', NEW_UNREAD_MESSAGES, (currentChatRoomUnreadMessages)
   const rootState = sbp('state/vuex/state')
   rootState.chatroom.unreadMessages = currentChatRoomUnreadMessages
 })
+
+export {}
```

Comment:
..What's this? (Weirdness = comment or remove)

Reply:
That's an empty export declaration and means that there are no exports. This is needed because esbuild will add `export default require_sw_primary()` to `sw-primary.js`, which is a syntax error as `sw-primary.js` is loaded as a classic worker (no ESM).

It wasn't needed before because this seems to happen after adding `"type": "commonjs"` to `package.json`, which is supposed to be the default but apparently it's not really a default, as it affects things like this (esbuild), and it had to be added because trying to use the 'default' value (i.e., `commonjs`) actually results in Node trying to guess whether a file is ESM or CommonJS (and it incorrectly guesses ESM due to imports).